masonry/src/event_loop_runner.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// We use allow because expect(missing_docs) is noisy with rust-analyzer.
```

Comment:
Can you clarify what this means? I'm not seeing any noise if I change this to `expect`

Reply:
I don't see a warning any more either. I guess they fixed whatever bug there was.